go/vt/vtctl/vtctl.go

```diff
@@ -118,6 +118,7 @@ import (
 	"vitess.io/vitess/go/vt/wrangler"
 
 	tabletmanagerdatapb "vitess.io/vitess/go/vt/proto/tabletmanagerdata"
+	"vitess.io/vitess/go/vt/proto/topodata"
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
```

Comment:
We shouldn't import the same package with 2 aliases. Could we change the usage of `topodata` to `topodatapb`

Reply:
Good catch. That went away now that I'm using the parseTabletType function.